src/views/SalesInfo/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Bar, BarChart, ErrorBar, Tooltip, XAxis, YAxis } from 'recharts';
+import { colors } from 'components/theme';
+
+import { ReactComponent as BuyWpoktSvg } from 'assets/icons/buy_wpokt_button.svg';
+
```

Comment:
Why do we have the spaces here? Is this consistent across the whole codebase?

Reply:
I have the spaces just to separate categories of imports. It's consistent between all files, but u think I should instead remove spaces between all imports in all files?

Reply:
No hard preference—just shooting for consistency. ;)